octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -443,6 +445,24 @@ $(function () {
         };
 
         self._handle_session_expired = function (triggerUrl) {
+            if (self.isCurtainOpened > 0 && self.triggerAnalytics) {
```

Comment:
Please change as we just discussed: Count 401 errors for 10 sec and then submit the result in 1 request to the backend.